src/lib/imports.ts

```diff
@@ -0,0 +1,40 @@
+export function getImportInsertion(
+  existingText: string,
+  importPath: string,
+  modulesToImport: string[],
+) {
+  const importRegex = new RegExp(`(import {.*?)\\s+} from "${importPath}"`)
```

Comment:
Possibly controversial: detect imports via regex rather than using a babel traversal. The idea here was that sometimes we need to scan the current file for the import, and sometimes the separate file for imports, and this was more flexible across both cases. Open to feedback.

Reply:
Not a problem to me! It's simple enough and can cover majority of cases. If users complains on corner cases, we can switch to something like [es-module-lexer](https://classic.yarnpkg.com/en/package/es-module-lexer) then!